ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/sync/BlockPropagationManagerTest.java

```diff
@@ -561,4 +566,25 @@ public void shouldNotImportBlocksThatAreAlreadyBeingImported() {
 
     verify(ethScheduler, times(1)).scheduleSyncWorkerTask(any(Supplier.class));
   }
+
```

Comment:
Idea: Would it test any addition workflow (I'm not sure offhand) if you added a third node?  ie, create the block on first node like currently.  have node 2 process everything with native (non-test code), and wait for the message to arrive or process on 3rd node?